iphone/Classes/TiUIScrollableViewProxy.m

```diff
@@ -73,9 +73,6 @@ -(NSUInteger)viewCount
 
 -(void)setViews:(id)args
 {
-#if defined(TI_USE_AUTOLAYOUT) || defined(TI_USE_KROLL_THREAD)
```

Comment:
This line was a critical issue for the following tickets. can you make sure that your change didn't affect these tickets when RUN-ON-MAIN-THREAD = FALSE and AUTO-LAYOUT = FALSE.
https://jira.appcelerator.org/browse/TIMOB-20493
https://jira.appcelerator.org/browse/TIMOB-20218
https://jira.appcelerator.org/browse/TIMOB-20199
https://jira.appcelerator.org/browse/TIMOB-20196
https://jira.appcelerator.org/browse/TIMOB-20138
https://jira.appcelerator.org/browse/TIMOB-19986
https://jira.appcelerator.org/browse/TIMOB-19908